fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -472,13 +471,7 @@ public static String export() {
         RequestLog current = RLOG.get();
         record(new Durations(current.aggregateDurations()));
         record(new Threads(current.threadIds));
-        try {
-            return current.mapper.writeValueAsString(current.info);
-        } catch (JsonProcessingException jpe) {
-            String msg = String.format("Exporting mega log line with id: '%s' to JSON failed.", current.logId);
-            LOG.warn(msg, jpe);
-            return msg;
-        }
+        return LogFormatterProvider.getInstance().write(current.info);
```

Comment:
We should make sure to remove the `mapper` field from this class, since it's no longer necessary.